docs/configure_your_resource.md

```diff
@@ -61,3 +63,33 @@ $ bin/console sylius:debug:resource 'App\Entity\book'
 ```
 
 By default, it will have the `app.book` alias in Sylius resource which is a concatenation of the application name and the resource name `{application}.{name}`.
+
+## Advanced configuration
+
+### Configure the resource name
```

Comment:
I suppose we can get rid of one of these titles (especially as the latter is clearly dependent, but is presented on the same level in the [list above](https://github.com/Sylius/SyliusResourceBundle/pull/687/files#diff-5f940b00cf239eec51664ad8d16bf299563a07ff60252d059225bf6df89765b5R8-R9)) 🖖

Reply:
Nevermind, I just saw [this PR](https://github.com/Sylius/SyliusResourceBundle/pull/689/) that changes my opinion 😄